maintenance/scripts/setup.sh

```diff
@@ -15,19 +15,16 @@ php artisan config:clear
 chown -R www-data ./storage ./vendor
 chmod -R a+r,a+w ./storage ./vendor ./bootstrap/cache
 
-# setup frontend workspace
-cd /var/www/html/frontend
-npm install
-
 # copy out new .env files
 cd /var/www/html/frontend
 cp ./talentsearch/.env.example ./talentsearch/.env --preserve=all
-cp ./indigenousapprenticeship/.env.example ./indigenousapprenticeship/.env --preserve=all
 cp ./admin/.env.example ./admin/.env --preserve=all
 
-# build projects
-cd /var/www/html/frontend
-npm run codegen --workspaces
-npm run intl-compile --workspaces
-npm run dev --workspaces --if-present # common workspace does not have dev script
+cd /var/www/html
+cp ./apps/web/.env.example ./apps/web/.env --preserve=all
 
+# build projects
+cd /var/www/html
+npm i
```

Comment:
Can we use `npm install` instead?  Verbose commands are easer to understand in scripts.

Reply:
Updated in [use explicit install command](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5173/commits/a5d60daeb8158acf22eeccbc8ba13bf37e3955ee)